src/components/ui/ContextMenuDropdown.tsx

```diff
@@ -329,15 +332,24 @@ export default function ContextMenuDropdown({
     }))
   }))
 
+  const buttonClass = useMemo(() => {
+    return mergeClasses('rounded-full before:rounded-full', size === 'small' ? 'w-9' : size === 'large' ? 'w-11' : 'w-10', className)
```

Comment:
I think the context menu should have an option to not use the button style at all. And when using the button style we should use the same radius as the default icon btn.

For example we have a context menu on the lazy book card
<img width="256" height="255" alt="image" src="https://github.com/user-attachments/assets/f78b05d2-20b4-449b-9bbb-a7981c5ca3de" />

And we have on the library item page
<img width="417" height="138" alt="image" src="https://github.com/user-attachments/assets/c9e80476-f219-4c08-a918-c2ac897e2dc1" />

That is using the horizontal more icon for no particular reason. It would be fine with `more_vert` also.